### DISCUSSION TITLE
May I have more information about sudo

### DISCUSSION THREAD

**Discussion by Qssaf** [EssentialsX/Essentials#5924] [Q&A] [answered]
I am new to the essentials plugin and I need to use /sudo for a staff job that I have, I searched the wiki but I don't see what are the arguments that I can add, I only found c: which makes the player send a message.
And also I want to know is there a way to be able to sudo console?

**Comment by pop4959** [comment, marked as answer]
Sudo allows you to run a command for another player. The `c:` option instead forces a chat message to be sent for that player. There are no other options for sudo, and you cannot sudo as the console to execute arbitrary commands.

**Reply by Qssaf** [reply]
Okay, thank you for the info :)
